genai-perf/README.md

```diff
@@ -556,6 +556,31 @@ The number of tokens in each prefix prompt. This value is only used if
 being concatenated, the number of tokens in the final prompt may be off by one.
 (default: `100`)
 
+##### `--image-width-mean <int>`
+
+The mean width of images when generating synthetic image data.
```

Comment:
Do you think it'd be useful to specify that you are measuring in pixels (I'm assuming that's the unit)?

Reply:
Yeah, I think it would be clearer for the reader. Mentioned pixel as unit in the desc.